modules/dnn/test/test_tf_importer.cpp

```diff
@@ -143,7 +143,9 @@ TEST_P(Test_TensorFlow_layers, Convolution3D)
 
 TEST_P(Test_TensorFlow_layers, padding)
 {
-    runTensorFlowNet("padding_valid");
+    runTensorFlowNet("padding_valid", false,
+        (target == DNN_TARGET_CUDA_FP16) ? 2 : 0.0,
+        (target == DNN_TARGET_CUDA_FP16) ? 2.1 : 0.0);
```

Comment:
Looks like a bug in the implementation. Too big difference for zero paddings.

Reply:
The test network for `padding_valid` uses ELU activation. The activation is the source of the error (there is a test on Conv_Elu which also has high errors).

```
[ RUN      ] Test_TensorFlow_layers.padding/1, where GetParam() = CUDA/CUDA_FP16
padding valid:
[CUDA] Convolution conv2d_3/convolution
[CUDA] ELU conv2d_3/Elu
Ref. Range: -0.980401 0.123652
padding spatial:
[CUDA] Convolution conv2d/convolution
Ref. Range: -2.53209 1.58402
mirror_pad:
[CUDA] Padding MirrorPad
Ref. Range: -3.00851 3.45245
keras_pad_concat:
[CUDA] Convolution keras_pad_concat_conv/Conv2D
[CUDA] Const keras_pad_concat/Variable
[CUDA] Concat keras_pad_concat/concatenate/concat
Ref. Range: -1.31247 1.212
[       OK ] Test_TensorFlow_layers.padding/1 (32 ms)
```

Reply:
Bug fixed in #16045. Custom thresholds removed.